conf/symbiflow/environment.yml

```diff
@@ -1,15 +1,16 @@
 name: symbiflow-env
 channels:
   - defaults
-  - symbiflow
+  - litex-hub
 dependencies:
-  - symbiflow-yosys=0.8_6021_gd8b2d1a2=20200708_083630
-  - symbiflow-yosys-plugins=1.0.0.7_0174_g5e6370a=20201012_171341
-  - symbiflow-vtr=8.0.0.rc2_5415_gd6d69ff92=20201120_180018
-  - prjxray-tools
-  - prjxray-db
-  - python=3.8
+  - litex-hub::capnproto-java=0.1.5_0012_g44a8c1e=20201104_165332
+  - litex-hub::gcc-riscv64-elf-newlib=9.2.0=20201119_154229
+  - litex-hub::symbiflow-yosys-plugins=1.0.0_7_g59ff1e6_23_g3a95697_17_g00b887b_0194_g40efa51=20201120_145821
+  - litex-hub::prjxray-tools=0.1_2697_g0f939808=20201120_145821
+  - litex-hub::prjxray-db=0.0_0239_gd87c844=20201120_145821
+  - litex-hub::vtr=v8.0.0_3011_gb0223dc59=20201202_112618
```

Comment:
Is this supposed to be synchronized with symbiflow-arch-defs somehow? This is not the same as the version used by symbiflow-arch-defs (currently d912bdb88)

Reply:
Currently, in CI we are using the environment.yml coming from the symbiflow-arch-defs install package, and not this one. I am unsure how to deal with the update of this static environment.yml, for now it needs to be manual.

To override which environment variable to use (at least when building in conda) you can do:
```
ENVIRONMENT_FILE=<custom_env> make env
```

the default points at the one in `conf/symbiflow/environment.yml`.